### PR TITLE
Documentation/11/update gh pages

### PR DESCRIPTION
**Pull request by helen-d-little**
Reorganised some documents, not complete.
Needs to be pushed to main before visible on ghpages
Nothing that should be NATS sensitive.

### CODE REVIEW

File: intro/about.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+## BlueSky
+[BlueSky](https://github.com/project-bluebird/bluesky) is the currently supported open-source simulator.
+
+
+## Aviary
+[![Build Status](https://travis-ci.com/project-bluebird/aviary.svg?branch=develop)](https://travis-ci.com/alan-turing-institute/aviary)
+
+[Aviary](../reference/aviary.md) is an air traffic scenario generation package developed as part of the BlueBird project. It aims to make available a large number of simple (but non-trivial) air traffic control scenarios suitable for the training of an artificial agent via reinforcement learning. The package also includes a set of evaluation metrics for measuring performance in the air traffic control challenge.
```
Comment (radka-j) [inline intro/about.md]: I'd remove reinforcement learning from this section - we provide platform for agent training, it's up to people what specific method they want to use

File: intro/install.md [hunk 2]
```diff
@@ -1,3 +1,154 @@
-# Installation instructions
+# Installation Instructions
+With an air traffic control (ATC) simulator (e.g., BlueSky) and BlueBird running, one can observe and interact with the simulation via BlueBird using Python (PyDodo), R (rdodo) or Twitcher. The Aviary package allows one to synthetically generate ATC scenarios of increasing complexity to train agents on and provides metrics to score performance.
+
+## Quick Start Using Simurgh
```
Comment (radka-j) [inline intro/install.md:4]: Should we (can we) move the example files to this repo and just get rid of simurgh entirely? The point of simurgh was to be the over-arching docs repo and the only thing it contains are the docker files and the example notebook. Ideally those would be all here and we can get rid of it. That said, I understand that this repo launches GitHub page, so maybe we don't want to ask people to have to clone this repo and *do* want a link to some other repo to clone. In that case, maybe we just rename and clean up simurgh. Which do you think is better?

File: reference/aviary.md [hunk 3]
```diff
@@ -2,22 +2,23 @@
 
 [![Build Status](https://travis-ci.com/project-bluebird/aviary.svg?branch=develop)](https://travis-ci.com/alan-turing-institute/aviary)
 
-Aviary is an air traffic scenario generation package developed as part of the [Simurgh](https://github.com/alan-turing-institute/simurgh) project. It aims to make available a large number of simple (but non-trivial) air traffic control scenarios suitable for the training of an artificial agent via reinforcement learning. The package also includes a set of evaluation metrics for measuring performance in the air traffic control challenge.
+Aviary is an air traffic scenario generation package developed as part of the BlueBird project. It aims to make available a large number of simple (but non-trivial) air traffic control scenarios suitable for the training of an artificial agent via reinforcement learning. The package also includes a set of evaluation metrics for measuring performance in the air traffic control challenge.
```
Comment (radka-j) [inline reference/aviary.md:5]: again, i'd remove mention of RL here

**Review by radka-j — COMMENTED** [review]
Looks good!! :)

File: intro/about.md (hunk 1 above)
Comment (helen-d-little) [reply intro/about.md]: Yup updated text and will put on next push (this afternoon)

File: intro/install.md (hunk 2 above)
Comment (helen-d-little) [reply intro/install.md:4]: Yes I agree - I think there is a lot of overlap between what simurgh does and what the ghpages will do.
I think copying over the contents of simaugh would make sense, as long as there is somewhere sensible for the docker files to go. So I'm not sure what the best option would be. I think the docs repo is public as it has to be in order for the pages to be launched so there's no reason that I can think of to not have them here, but there could be good reason to leave them be?

File: reference/aviary.md (hunk 3 above)
Comment (helen-d-little) [reply reference/aviary.md:5]: Oddly enough I had changed it to BlueBird but obviously not in this push! RL removed based on above

**Pull request merged**